serverstuff/server.js

```diff
@@ -151,5 +151,9 @@ io.on('connection', socket => {
 });
 
 http.listen((port = 3000), function () {
+  if(!fs.existsSync(__dirname + '/uploads')) {
+    console.log('Uploads-folder not found, recreating.')
+    fs.mkdirSync(__dirname + '/uploads');
+  }
```

Comment:
Burde det ikke kjøre ved oppstart, i stedet for på den første requesten?

Reply:
Det er på oppstart

Reply:
Ååå